pkg/tfvars/baremetal/baremetal_test.go

```diff
@@ -162,7 +162,7 @@ func TestMastersSelectionByRole(t *testing.T) {
 	for _, tc := range cases {
 		t.Run(tc.scenario, func(t *testing.T) {
 
-			imageDownloader = func(baseURL, applicationName string) (string, error) {
+			imageDownloader = func(baseURL, applicationName string, sha256Checksum string) (string, error) {
```

Comment:
(commenting here just because it's a test file) it would be nice to have at least an it